05-FITS/io_fits_ascii_overview.ipynb

```diff
@@ -136,25 +143,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "keyname = 'CRPIX1'\n",
+    "extnum = 1\n",
+    "\n",
     "with fits.open('j94f05bgq_flt.fits.gz', mode='update') as f:\n",
-    "    print(\"crpix1: \", f[1].header['crpix1'])\n",
-    "    f[1].header['crpix1'] = 12  # Reassign the keyword\n",
-    "    print(\"Updated crpix1 :\", f[1].header['crpix1'])"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Opening a FITS file using the Python *with* statement\n",
-    "ensures that the file is closed without explicitely doing so."
+    "    value = f[extnum].header[keyname]\n",
```

Comment:
`header.get()` does not raise the KeyError IIRC and it also allows the specification of a default (which defaults to `None` if the keyword is not in the header). I would like us to show that as best practice.